website/docs/r/ses_identity_notification.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: "aws"
+page_title: "AWS: ses_identity_notification"
+sidebar_current: "docs-aws-resource-ses-identity-notification"
+description: |-
+  Setting SES Identity Notification
+---
+
+# ses_identity_notification
+
+Setting SES Identity Notification for a domain
```

Comment:
This resource can be used for non-domain SES identities such as email addresses. This can be `Resource for managing SES Identity Notifications` or similar.